cves/CVE-2013-2251.yml

```diff
@@ -103,9 +105,13 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: There were unit tests to check certain features of the DefaultAction
+          Mapper functionality, including unsafe redirects. However, the 
+          specific case of OGNL expression execution for the vulnerable 
+          parameters was not tested. I don't see tests for it in the patch
+          and the parameters that were tested after patch were deprecated.
```

Comment:
+2 because if the developers would have thought to add a unit test for this particular input, this vulnerability might have been avoided.

Reply:
Thank you

Points applied